src/components/CategoriesItem/CategoriesItem.jsx

```diff
@@ -3,29 +3,44 @@ import styles from "./CategoriesItem.module.css";
 import clsx from "clsx";
 import { setSelectedCategory } from "../../redux/categories/slice";
 
-function CategoriesItem({ id, title, option = "category" }) {
+function CategoriesItem({ id, title, description, option = "category", onClick }) {
 
     const image = `/src/assets/categories/${title}.webp`;
 
     const dispatch = useDispatch();
 
     const handleClick = () => {
-        if (option === "all") {
+        if (option === "All") {
+            onClick();
+        } else if (option === "Show") {
             dispatch(setSelectedCategory({
                 id: null,
-                title
+                title,
+                description 
             }));
         } else {
             dispatch(setSelectedCategory({
                 id,
-                title
+                title,
+                description
             }));
         }
     }
 
+    const optionClassMap = {
+        All: {
+            container: styles.categoriesContainerAll,
+            title: styles.categoriesTitleAll,
+        },
+        Show: {
+            container: styles.categoriesContainerShow,
+            title: styles.categoriesTitleShow,
+        },
+      };
+
     return (
         option === "category" ? (
-            <li className={styles.categoriesContainer}>
+            <li className={styles.categoriesContainer} onClick={handleClick}>
```

Comment:
on handler on li, u should add include button inside li and handle on click button inside li